### ISSUE TITLE
[3.55] `type GameObjectConfig` is incorrect now

### ISSUE THREAD

**Issue by vforsh** [photonstorm/phaser#5713]
## Version
3.55

## Description
Something went wrong with Typescript definitions generation so `type GameObjectConfig` looks like this now:

```
type GameObjectConfig = {
                undefined: any;
                undefined: any;
                /**
                 * The depth of the GameObject.
                 */
                depth?: number;
                /**
                 * The horizontally flipped state of the Game Object.
                 */
                flipX?: boolean;
                /**
                 * The vertically flipped state of the Game Object.
                 */
                flipY?: boolean;
                /**
                 * The scale of the GameObject.
                 */
                scale?: number | object;
                /**
                 * The scroll factor of the GameObject.
                 */
                scrollFactor?: number | object;
                undefined: any;
                undefined: any;
                undefined: any;
                /**
                 * The origin of the Game Object.
                 */
                origin?: number | object;
                /**
                 * The scale mode of the GameObject.
                 */
                scaleMode?: number;
                /**
                 * The blend mode of the GameObject.
                 */
                blendMode?: number;
                /**
                 * The visible state of the Game Object.
                 */
                visible?: boolean;
                /**
                 * Add the GameObject to the scene.
                 */
                add?: boolean;
            };
```

**Comment by photonstorm** [comment]
Yes, sadly PR #5683 from @karbassi was full of typos: `{(number|object}}` note the error } instead of ), but I didn't notice until it was too late as it didn't throw any errors during TS def generation or testing! This is now fixed in the master branch but you'll have to regenerate the defs to get them. I may release a 55.1 to address this.

**Comment by karbassi** [comment]
Oh! I'm sorry for the typos. My testing didn't throw any errors either. Let me know if you'd like me to fix it.

**Comment by photonstorm** [comment]
@karbassi yeah don't worry, I sorted it already. Was just annoyed at myself that I didn't spot it - and quite surprised it didn't cause a TS test error! But oh well, such is life.

**Comment by vforsh** [comment]
> Yes, sadly PR #5683 from @karbassi was full of typos: `{(number|object}}` note the error } instead of ), but I didn't notice until it was too late as it didn't throw any errors during TS def generation or testing! This is now fixed in the master branch but you'll have to regenerate the defs to get them. I may release a 55.1 to address this.

yes, `npm run tsgen` against master branch did work perfectly

thanks!

**Comment by photonstorm** [comment]
Fixed in 3.55.1.
